README.md

```diff
@@ -884,4 +884,4 @@ Currently mobile is not supported. However, there are plans to add touch support
 
 ## Author / maintainer
 
-Alex Reardon - [@alexandereardon](https://twitter.com/alexandereardon) - areardon@atlassian.com
\ No newline at end of file
+Alex Reardon - [@alexandereardon](https://twitter.com/alexandereardon) - areardon@atlassian.com
```

Comment:
Nah all good - sorry for the trouble too! Please do not hesitate to submit other PR's going forward 👍